Jenkinsfile

```diff
@@ -51,6 +52,21 @@ def run_full_build()
   return true
 }
 
+def set_up_env_for_linux_pipenv(steps)
+{
+  withEnv([
+    "LC_ALL=C.UTF-8",
+    "LANG=C.UTF-8"])
+  {
+    steps()
+  }
+}
```

Comment:
I'm not sure if we should have this here or in the image. Here is less invasive, but then again not having a UTF-8 locale feels very last-century on the part of the image. Thoughts?

Reply:
I have a mild preference for putting it in the image to be honest

Reply:
I'll give it a go.

Reply:
Done.